### ISSUE TITLE
Macos - Android Build FAILURE: Build failed with an exception.

### ISSUE THREAD

**Issue by mribbons** [socketsupply/socket#401]
@lamiazar 

this what I'm getting hen I run emulator on Mac
```
mia@mias-Air app234 % ssc build -r -o --platform=android-emulator
[=======================================] 100% Computing updates...             
All SDK package licenses accepted.

• package prepared +4402ms
[=======================================] 100% Computing updates...             

Starting a Gradle Daemon, 1 incompatible Daemon could not be reused, use --status for details

> Configure project :app
WARNING:The option setting 'android.experimental.legacyTransform.forceNonIncremental=true' is experimental.
The current default is 'false'.

> Task :app:processDevDebugMainManifest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDevDebugMainManifest'.
> com.android.manifmerger.ManifestMerger2$MergeFailureException: Error parsing /Users/mia/app234/build/android-emulator/app/src/main/AndroidManifest.xml

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4s
6 actionable tasks: 6 executed
• ERROR: failed to invoke ./gradlew :app:bundleDebug --warning-mode all command. +6218ms
```

**Comment by mribbons** [comment]
@lamiazar,

This bug should be fixed on `master`, please post the output from:
```bash 
cat $(find build/android/app/src -name AndroidManifest.xml)
```

**Comment by lamiazar** [comment]
> This bug should be fixed on master, please post the output from:
`cat $(find build/android/app/src -name AndroidManifest.xml)`

```
cat $(find build/android/app/src -name AndroidManifest.xml)
find: build/android/app/src: No such file or directory
```

**Comment by mribbons** [comment]
Notes from DM:
```
cat $(find build/android/app/src -name AndroidManifest.xml)
# should be:
cat $(find build/android-emulator/app/src -name AndroidManifest.xml)
```
It contained `false` which comes from an old version of `ssc`.
The solution is to update `ssc`, and remove `allow_cleartext = false` from `socket.ini`.

We still have an issue with `avdmanager` not being found:
```bash
BUILD SUCCESSFUL in 23s
140 actionable tasks: 86 executed, 54 up-to-date
sh: avdmanager: command not found
# ...
adb: no devices/emulators found
• ERROR: failed to install APK to Android Emulator (adb) +178ms
```

**Comment by mribbons** [comment]
```bash
Execution failed for task ':app:processDevDebugMainManifest'.
> com.android.manifmerger.ManifestMerger2$MergeFailureException: Error parsing /Users/mia/app234/build/android-emulator/app/src/main/AndroidManifest.xml
```
Confirming that this is resolved on `master`.

```
• ERROR: failed to install APK to Android Emulator (adb) +178ms
```
Will be fixed in #404
